mocks/hit-endpoints.js

```diff
@@ -0,0 +1,99 @@
+const axios = require('axios').default;
+
+const settings = require('./settings');
+
+sendRequestToAllEndpoints();
+
+async function sendRequestToAllEndpoints() {
+  const routeResponse = await axios.get(
+    'http://localhost:3110/api/mock/routes'
+  );
+
+  for (let i = 0; i < routeResponse.data.length - 1; i++) {
+    const entry = routeResponse.data[i];
+
+    if (entry.id in CUSTOM_REQUESTS) {
+      CUSTOM_REQUESTS[entry.id]();
+      continue;
+    }
+
+    // Regex to replace all url parameters, like for example '/:something/' becomes '/1/'.
+    const paramReplacingPattern = new RegExp('(?<=\/):.+?(?=\/|$)', 'g'); // prettier-ignore
+    // reason: It will remove all escape sequences in the regular expression which is undesired.
```

Comment:
is this considering paramReplacingPattern or requestUrl

Reply:
Beetje rare naam inderdaad. Heb het aangepast. De prettier-ignore is voor de regex. Heb ook de reason wat explicieter gemaakt. Begrijp ik je vraag dan goed?

Reply:
ja klopt goed!